src/main/scala/org/broadinstitute/dsp/HelmInterpreter.scala

```diff
@@ -61,6 +61,25 @@ class HelmInterpreter[F[_]](concurrencyBound: Semaphore[F])(implicit logger: Log
       _ <- Kleisli.liftF(translateResult("helm list", "ok"))
     } yield ()
 
+  def updateAndPullChart(chartName: ChartName,
+                         chartVersion: ChartVersion,
+                         destDir: String): Kleisli[F, AuthContext, Unit] =
```

Comment:
I don't think the `AuthContext` is used here, right? Think we could make this return type `Kleisli[F, Unit, Unit]`